src/commands/Poscon.ts

```diff
@@ -98,22 +98,22 @@ export class Poscon {
                         },
                         {
                             name: "Latitude",
-                            value: posconClient.position.lat.toString(),
+                            value: posconClient.position?.lat.toString() ?? "-",
```

Comment:
```suggestion
                            value: posconClient.position?.lat?.toString() ?? "N/A",
```

`-` seems just random. Maybe like this? (for others also)

Reply:
Or maybe `unknown`.